validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java

```diff
@@ -101,10 +103,17 @@ private static void addLocalValidatorSource(
       final SlashingProtector slashingProtector,
       final AsyncRunner asyncRunner,
       final List<ValidatorSource> validatorSources) {
-    if (config.getValidatorKeystorePasswordFilePairs() != null) {
+    KeyStoreFilesLocator keyStoreFilesLocator =
+        new KeyStoreFilesLocator(config.getValidatorKeys(), File.pathSeparator);
+    if (keyStoreFilesLocator.parse() != null) {
```

Comment:
I don't think we want to parse here - just check if `config.getValidatorKeys()` is null instead.  If it is null we can skip creating the `KeyStoreFilesLocator` entirely.
```suggestion
    if (config.getValidatorKeys() != null) {
        KeyStoreFilesLocator keyStoreFilesLocator =
            new KeyStoreFilesLocator(config.getValidatorKeys(), File.pathSeparator);
```